devtools/gradle/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     implementation "io.quarkus:quarkus-platform-descriptor-resolver-json:${version}"
     implementation "io.quarkus:quarkus-development-mode:${version}"
 
+    testImplementation 'org.mockito:mockito-core:3.2.4'
```

Comment:
Is this related to the purpose of the PR?

Reply:
This is for the test. There are too many nested objects in Gradle :)

Reply:
Perhaps not as part of this PR, I think we should extract specific versions to the properties file (`gradle.properties`) that would also make it easier to verify/identify differences between the maven and gradle projects. We could also generate the properties file from a maven plugin, as an option.